django/db/backends/oracle/compiler.py

```diff
@@ -42,7 +42,36 @@ def as_sql(self, with_limits=True, with_col_aliases=False, subquery=False):
 
 
 class SQLInsertCompiler(compiler.SQLInsertCompiler, SQLCompiler):
-    pass
+    def as_sql(self):
```

Comment:
Oracle specifies restrictions on multi-table inserts:

>   Restrictions on Multitable Inserts Multitable inserts are subject to the following restrictions:
>    -  You can perform multitable inserts only on tables, not on views or materialized views.
>    -  You cannot perform a multitable insert into a remote table.

So if we use this, the result would be "Django cannot do `bulk_create()` if the model is stored in a view on Oracle". I'd rather avoid this limitation unless there are very substantial benefits to be had.